pyret/visualizations.py

```diff
@@ -240,10 +243,9 @@ def play_sta(sta, repeat=True, frametime=100, cmap='seismic_r', clim=None):
     # Set up the figure
     fig = plt.figure()
     plt.axis('equal')
-    ax = plt.axes(xlim=(0, X.shape[1]), ylim=(0, X.shape[2]))
-    img = plt.imshow(initial_frame)
-    ax.set_xticks([])
-    ax.set_yticks([])
+    extent = (0.0, X.shape[1] * dx, 0.0, X.shape[2] * dx)
```

Comment:
Can we use a more descriptive word than 'extent'? Maybe, 'spatial_range'?

Reply:
Sure, `spatial_range` is fine. I just used `extent` since it's the kwarg to `ax.imshow()`.